driver/pelican/main.go

```diff
@@ -142,15 +160,40 @@ func main() {
 				}
 				fmt.Printf("%s %+v\n", currentPelican.name, status)
 
-				po, err := bw2.CreateMsgPackPayloadObject(bw2.FromDotForm(TSTAT_PO_DF), status)
-				if err != nil {
-					fmt.Printf("Failed to create msgpack PO: %v", err)
-					done <- true
+				// Nil status indicates there is no sufficiently recent data
+				if status != nil {
+					po, err := bw2.CreateMsgPackPayloadObject(bw2.FromDotForm(TSTAT_PO_DF), status)
+					if err != nil {
+						fmt.Printf("Failed to create msgpack PO: %v", err)
+						done <- true
+					}
+					currentIface.PublishSignal("info", po)
 				}
-				currentIface.PublishSignal("info", po)
 				time.Sleep(pollInt)
 			}
 		}()
 	}
 	<-done
 }
+
+func getTimeZone(sitename, username, password string) (string, error) {
```

Comment:
I don't think we want this logic in `main.go`. Rather, we should keep all interaction with Pelican's API within `pelican.go`.

Seems like you have two choices:
  1. Expose a `GetTimezone` function from `pelican.go` that can be called from `main.go`
  2. Fold the timezone discovery logic into the `DiscoverPelicans` function, so this handles figuring out the site's timezone as well as finding the pelican units present at that site.